### PR TITLE
DCOS_OSS-1780: Update Tables to Match UI Kit

### PR DESCRIPTION
**Pull request by ashenden**
The DC/OS UI Kit has made a number of improvements and adjustments to the appearance of the Tables. The majority of these changes are now represented by default in CNVS. This PR removes all redundant or conflicting styles from the DC/OS UI relating to tables. This PR also adjusts class names of tables to mirror expectations for the display offered by CNVS.  Lastly, all column headers, previously defined in ALL CAPS in React code, have been converted to Title case and the UI now uses CSS to transform the text.

### CODE REVIEW

File: src/styles/index.less [hunk 1]
```diff
@@ -102,18 +102,18 @@
 
 // Tables
 
-@import 'content/tables/variables.less';
-@import 'content/tables/styles.less';
+// @import 'content/tables/variables.less';
```
Comment (MatApple) [inline src/styles/index.less]: @ashenden let's just remove the commented out lines.

File: plugins/nodes/src/js/pages/nodes/NodeDetailTab.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ class NodeDetailTab extends React.Component {
                 Active
               </ConfigurationMapLabel>
               <ConfigurationMapValue>
-                {StringUtil.capitalize(node.active.toString().toLowerCase())}
+                s{StringUtil.capitalize(node.active.toString().toLowerCase())}
```
Comment (juliangieseke) [inline plugins/nodes/src/js/pages/nodes/NodeDetailTab.js]: is that `s` supposed to be there?

File: src/styles/index.less (hunk 1 above)
Comment (MatApple) [reply src/styles/index.less]: @ashenden what's the status on these files? Do they need to be removed?

**Comment by ashenden** [comment]
@MatApple and @juliangieseke all tests passed, and from my POV this is ready for your final review and merge.

Please don't forget this PR (https://github.com/mesosphere/dcos-ui-plugins-private/pull/688) which has related fixes.

**Pull request merged**